webpack.config.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const path = require("path");
+
+module.exports = {
+    entry: {
+        demo: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/demo/index.js")],
+        parser: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/parser/index.js")]
+    },
+    output: {
+        path: path.join(__dirname, "assets/build/js"),
+        filename: "[name].js"
```

Comment:
Do we need content hash here to avoid browsers cache issue?

Reply:
That's a good question. Going off [this blog post by Netlify](https://www.netlify.com/blog/2017/02/23/better-living-through-caching/), it seems like we shouldn't need to. This seems to be the [same conclusion the Babel team came to](https://github.com/babel/website/blob/master/webpack.config.js#L10-L11).